src/com/serotonin/mango/web/dwr/beans/ImportTask.java

```diff
@@ -427,6 +430,13 @@ private void importDataSource(JsonObject dataSource) {
 					name == null ? "(undefined)" : name);
 		else {
 			DataSourceVO<?> vo = dataSourceDao.getDataSource(xid);
+			if (vo != null) {
+			    //Check the enable/disable is changed. And to test when don't have enable property when import data
+                boolean importedDsEnabled = dataSource.getBoolean("enabled");
```

Comment:
this variable importedDsEnabled is used only once.From my point of view we can put in condition code like
vo.isEnabled() != (dataSource.getBolean("enabled")

Reply:
My code facilitates debugging